src/univalue/include/univalue.h

```diff
@@ -170,8 +171,7 @@ static inline bool jsonTokenIsValue(enum jtokentype jtt)
     default:
         return false;
     }
-
-    // not reached
+    assert(false);
```

Comment:
Any reason to silence compiler warnings in a refactoring commit?

Reply:
Can be a separate commit if you'd prefer

Reply:
Will drop